marketplace_builder/pages/getting-started/users/logging-out-authenticated-user.liquid

```diff
@@ -0,0 +1,71 @@
+---
+converter: markdown
+metadata:
+  title: Logging Out an Authenticated User
+slug: get-started/users/logging-out-authenticated-user
+---
+
+This guide will help you create an authentication form to log out authenticated users. 
+
+## Requirements
+
+To follow the steps in this tutorial, you should be familiar with the required directory structure for your code base, and understand the concept of pages and users. 
+
+* [Required Directory Structure](/how-platformos-works/platformos-components)
+* [Pages](/get-started/pages/pages)
+* [Users](/get-started/users/users) 
+
+## Steps
+
+Logging out an authenticated user is a two-step process:
+
+1. Create log out form
+2. Embed the log out form
+
+### Step 1: Create log out form
+
+Create a form `log_out.liquid` in the `marketplace_builder/form_configurations` directory:
+
+{% raw %}
+
+```liquid
+---
+name: log_out
+resource: Session
+---
+
+{% form method: 'delete' %}
+  {% submit 'Log Out' %}
```

Comment:
I think its a better idea to promote submitting form as users want to without the helper which introduces additional learning curve.

This can be presented as:
`<input type="submit" value="Log Out" />` or `<button>Log Out</button>`

And it should be understood by everyone with basic html knowledge